doc/source/rayserve/overview.rst

```diff
@@ -0,0 +1,197 @@
+RayServe: Scalable and Programmable Serving
+===========================================
+
+.. image:: logo.svg
+    :align: center
+    :height: 250px
+    :width: 400px
+
+Overview
+--------
+
+RayServe is a scalable model-serving library built on Ray Core.
+
+RayServe allows you to:
```

Comment:
This wording is pretty awkward with the tenses of the bullets. Maybe `RayServe highlights:`?

Reply:
Or change first bullet to be a verb

Reply:
Changed this, lmkwyt